lib/roadie/rails/mailer.rb

```diff
@@ -1,9 +1,9 @@
 module Roadie
   module Rails
     module Mailer
-      def roadie_mail(options = {}, &block)
+      def roadie_mail(options = {}, custom_roadie_options = nil, &block)
         email = mail(options, &block)
-        MailInliner.new(email, roadie_options).execute
+        MailInliner.new(email, custom_roadie_options || roadie_options).execute
```

Comment:
I would prefer

```ruby
def roadie_mail(options = {}, roadie_options = self.roadie_options, &block)
```

Then the documentation would also show where the default value comes from straight in the method signature, and also that the provided arguments are replacing the method and not merging with it.

Reply:
I think we cannot use this signature because `self` points to the class this module is included in. It breaks specs.

Reply:
That is very strange. Default arguments should be evaluated in the instance context. I'll take a look.

Reply:
So the problem was that the integration tests used a `roadie_options` method with a `private` modifier. This disallows `self.roadie_options` as it has an explicit receiver.

I liked your final solution of `final_roadie_options` so I'm keeping that. :+1: